src/components/ChooseADate.jsx

```diff
@@ -127,46 +126,39 @@ class ChooseADate extends Component {
     return arrYear;
   }
 
-  //On lui dit ce qui va changer
-  componentDidMount() {
-    this.getGalerie(this.state.year, this.state.month, this.state.day);
-  }
-
   getGalerie(year, month, day) {
-    // Récupération de l'API et changement des 3 paramètres
+    // Retrieving the API and changing the 3 parameters
+    const api_key = process.env.REACT_APP_APIDAY_KEY;
```

Comment:
Just lol'd hard
You have a problem with naming your variables, you know that ? ^^